### PR TITLE
Run sfc_climo_gen utility on Hera

### LINKED ISSUE #788: Add a script to run `./util/sfc_climo_gen` on Hera
A user requested a script to run this utility on Hera.

Also, ensure both the new Hera script and the existing WCOSS2 script can find the new 'frac_oro' data (i.e., `./fix/orog/C384.mx025`)

### PR DESCRIPTION
**Pull request by GeorgeGayno-NOAA**
## DESCRIPTION OF CHANGES: 
Add script to run the sfc_climo_gen utility on Hera. Add common logic to a new
script.

## TESTS CONDUCTED: 
Tested on WCOSS2 and Hera

## DOCUMENTATION:
N/A

## ISSUE: 
Fixes #788

### CODE REVIEW

File: util/sfc_climo_gen/run.hera.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#------------------------------------------------------------
+# Run the sfc_climo_gen program stand-alone on Hera using
+# pre-exiting 'grid' and 'orography files.
+#
+# Set the configuration variables in sfc_gen.sh. Then
+# run this script as follows: 'sbatch $script'
+#------------------------------------------------------------
+
```
Comment (JeffBeck-NOAA) [inline util/sfc_climo_gen/run.hera.sh:11]: The pre-existing 'grid' and 'orography' files need to be staged in FIX_FV3=${BASE_DIR}/fix/orog/C${res}, right?  Can that info be added to the comment block above, including the names of the files?

File: util/sfc_climo_gen/run.hera.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#------------------------------------------------------------
+# Run the sfc_climo_gen program stand-alone on Hera using
+# pre-exiting 'grid' and 'orography files.
+#
```
Comment (JeffBeck-NOAA) [inline util/sfc_climo_gen/run.hera.sh:7]: 'orography', missing second quotation.

File: util/sfc_climo_gen/run.wcoss2.sh [hunk 3]
```diff
@@ -22,88 +23,6 @@ export APRUN_SFC="mpiexec -n 24 -ppn 24 -cpu-bind core"
...
-
-export input_sfc_climo_dir=${BASE_DIR}/fix/sfc_climo
-
-ulimit -a
-ulimit -s unlimited
-
-rm -fr $WORK_DIR $SAVE_DIR
-
-${BASE_DIR}/ush/sfc_climo_gen.sh
+$PBS_O_WORKDIR/sfc_gen.sh
```
Comment (JeffBeck-NOAA) [inline util/sfc_climo_gen/run.wcoss2.sh:27]: Why was the block of code below removed?  I assume we want to keep this functionality on WCOSS2, correct?

File: util/sfc_climo_gen/run.hera.sh (hunk 2 above)
Comment (GeorgeGayno-NOAA) [reply util/sfc_climo_gen/run.hera.sh:7]: Fixed.

File: util/sfc_climo_gen/run.wcoss2.sh [hunk 4]
```diff
@@ -2,9 +2,11 @@
 
 #------------------------------------------------------------
 # Run the sfc_climo_gen program stand-alone on WCOSS2 using
-# pre-exiting 'grid' and 'orography files.
+# pre-exiting 'grid' and 'orography files. See the
+# sfc_gen.sh script for details.
 #
```
Comment (JeffBeck-NOAA) [inline util/sfc_climo_gen/run.wcoss2.sh:7]: One more quotation here.

Comment (GeorgeGayno-NOAA) [reply util/sfc_climo_gen/run.wcoss2.sh:7]: Fixed. Thanks.

File: util/sfc_climo_gen/run.hera.sh (hunk 1 above)
Comment (GeorgeGayno-NOAA) [reply util/sfc_climo_gen/run.hera.sh:11]: Added more comments at 5baa903.

File: util/sfc_climo_gen/run.wcoss2.sh (hunk 3 above)
Comment (GeorgeGayno-NOAA) [reply util/sfc_climo_gen/run.wcoss2.sh:27]: The common logic - including the invocation of `sfc_climo_gen.sh` - was moved to new script `sfc_gen.sh`.  That new script is called from the hera and wcoss2 (and any future) driver scripts.

Comment (JeffBeck-NOAA) [reply util/sfc_climo_gen/run.wcoss2.sh:27]: Sorry, I got confused when comparing the wrong files.  Looks good!

**Review by JeffBeck-NOAA — APPROVED** [review]

**Pull request merged**